services/QuillCMS/app/controllers/responses_controller.rb

```diff
@@ -36,9 +36,11 @@ def create_or_increment
     if !response
       new_vals = transformed_new_vals(params_for_create)
       response = Response.new(new_vals)
-      if response.save
+      if !response.text.blank? && response.save
         AdminUpdates.run(response.question_uid)
         render json: response, status: :created, location: response
+      else
```

Comment:
I like that you noticed that this was missing and added it.

Reply:
:)